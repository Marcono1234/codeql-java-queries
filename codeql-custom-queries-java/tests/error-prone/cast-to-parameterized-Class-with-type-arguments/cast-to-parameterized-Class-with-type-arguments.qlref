./error-prone/cast-to-parameterized-Class-with-type-arguments.ql
