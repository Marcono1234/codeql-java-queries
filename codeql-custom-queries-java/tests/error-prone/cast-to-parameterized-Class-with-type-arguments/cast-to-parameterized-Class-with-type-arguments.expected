| Test.java:7:13:7:35 | (...)... | Casts to error-prone Class type Class<List<String>> |
| Test.java:8:13:8:30 | (...)... | Casts to error-prone Class type Class<List<T>> |
| Test.java:9:13:9:38 | (...)... | Casts to error-prone Class type Class<Map<?,Integer>> |
